nucliadb/nucliadb/ingest/orm/knowledgebox.py

```diff
@@ -91,25 +91,14 @@ async def get_config(self) -> Optional[KnowledgeBoxConfig]:
             return self._config
 
     @classmethod
-    async def delete_kb(cls, txn: Transaction, slug: str = "", kbid: str = ""):
+    async def delete_kb(cls, txn: Transaction, kbid: str):
```

Comment:
Can you check that backend doesn't need to delete by slug?

Reply:
Checked, they only use the kbid